core/src/main/java/com/coderanch/blackjack/Hand.java

```diff
@@ -113,4 +114,13 @@ private static int calculateBestScore(int minimumScore, int numberOfFreeAces) {
     boolean isBust() {
         return bestScore() > MAX_LEGAL_SCORE;
     }
+
+    /**
+     * Gets the cards of the hand.
+     *
+     * @return an {@code unmodifiableList} of cards.
```

Comment:
Use prose instead of code, unless you are referring to literals or method parameters.

`@return an unmodifiable list of cards.`